functests/1_performance/performance.go

```diff
@@ -49,11 +50,21 @@ const (
 	testPollInterval = 2
 )
 
+var snoTimeoutMultiplier time.Duration = 0
```

Comment:
I don't see this var init in other tests, is that intentional ?

Reply:
I wasn't sure where to declare it for the __performance package, so I chose that file.
do you have a suggestion for a better place to put it?

Reply:
here is fine if we keep the multiplier concept (which I'm not sold about yet)